frontend/packages/operator-lifecycle-manager/mocks.ts

```diff
@@ -737,6 +737,9 @@ export const operatorHubTileViewPageProps = {
       categories: ['messaging', 'streaming'],
       catalogSource: 'testing',
       catalogSourceNamespace: 'openshift-marketplace',
+      paidSubscription: undefined,
```

Comment:
Mocks should all be undefined?

Reply:
> Mocks should all be undefined?

The values are not used in the unit test, just the shape of the object. I can give them values but it makes no diff.

Reply:
ok great. focused for unit tests is good.